File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/adapters/holders/PicViewHolder.java
package com.task.paginationlist.presentation.piclist.views.adapters.holders;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.LinearLayout;
import android.widget.TextView;

import com.task.paginationlist.R;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Created by kirill on 13.07.17.
 */

public class PicViewHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.pic_container)
    public LinearLayout container;

    @BindView(R.id.image_place)
    public ImageView image;

    @BindView(R.id.image_date)
    public TextView date;

    public PicViewHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/task/paginationlist/presentation/piclist/views/adapters/holders/PicViewHolder.kt
package com.task.paginationlist.presentation.piclist.views.adapters.holders

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.ImageView
import kotlinx.android.synthetic.main.pic_list_item.view.*

/**
 * Created by kirill on 13.07.17.
 */

class PicViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var container = itemView.picContainer

    var image: ImageView = itemView.imagePlace

    var date = itemView.imageDate

}
